src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -0,0 +1,3394 @@
+        weakness L
+        move "Deep Sea Swirl", {
+          text "Shuffle your hand into your deck, then draw 8 cards."
+          energyCost C
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
+        move "Wave Splash", {
+          text "20 damage. "
+          energyCost W
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+      };
+      case MAREANIE_53:
+      return basic (this, hp:HP070, type:W, retreatCost:2) {
+        weakness L
+        move "Recover", {
+          text "Heal 30 damage from this Pokémon."
+          energyCost C
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
+        move "Poison Tentacles", {
+          text "20 damage. Your opponent’s Active Pokémon is now Poisoned."
+          energyCost W, C
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+      };
+      case TOXAPEX_54:
+      return evolution (this, from:"Mareanie", hp:HP130, type:W, retreatCost:3) {
+        weakness L
+        move "Recover", {
+          text "Discard an Energy from this Pokémon. If you do, heal all damage from this Pokémon."
+          energyCost C
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
+        move "Poison Whip", {
+          text "80 damage. Your opponent’s Active Pokémon is now Poisoned."
+          energyCost W, C, C
+          attackRequirement {}
+          onAttack {
+            damage 80
+          }
+        }
+      };
+      case DRACOVISH_55:
+      return evolution (this, from:"Rare Fossil", hp:HP150, type:W, retreatCost:3) {
+        weakness L
+        bwAbility "Primal Law", {
+          text "If this Pokémon is your Active Pokémon, your opponent can’t play any Pokémon from their hand to evolve their Pokémon."
+          actionA {
+          }
+        }
+        move "Hammer In", {
+          text "120 damage. "
+          energyCost W, C, C
+          attackRequirement {}
+          onAttack {
+            damage 120
+          }
+        }
+      };
+      case ARCTOVISH_56:
+      return evolution (this, from:"Rare Fossil", hp:HP150, type:W, retreatCost:2) {
+        weakness L
+        move "Hard Face", {
+          text "90 damage. During your opponent’s next turn, this Pokémon takes 60 less damage from attacks."
+          energyCost W, C, C
+          attackRequirement {}
+          onAttack {
+            damage 90
+          }
+        }
+        move "Cold Breath", {
+          text "130 damage. Your opponent’s Active Pokémon is now Asleep."
+          energyCost W, C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 130
+          }
+        }
+      };
+      case ELECTRIKE_57:
+      return basic (this, hp:HP060, type:L, retreatCost:1) {
+        weakness F
+        move "Thunderbolt", {
+          text "60 damage. Discard all Energy from this Pokémon."
```

Comment:
I believe damage for this attack is 50

[Electrike](https://www.pokebeach.com/news/0520/038168_P_RAKURAI.jpg)